src/firmware_new/boards/frankie_v1/io/drivetrain.h

```diff
@@ -1,29 +1,88 @@
 #pragma once
 
+#include "firmware/shared/math/matrix.h"
 #include "firmware_new/boards/frankie_v1/io/drivetrain_unit.h"
 
+typedef struct Drivetrain Drivetrain_t;
 /**
  * Initialize the IO layer for the Drivetrain
  *
  * NOTE: This does not take ownership of the given drive units. The driveunit's must
  *       remain valid until this function is called again.
  *
- * @param [in] front_left_drive_unit
- * @param [in] front_right_drive_unit
- * @param [in] back_left_drive_unit
- * @param [in] back_right_drive_unit
+ * @param front_left_drive_unit [in]
+
```

Comment:
nit: remove blank lines between `@params`

Reply:
Do you mean the single blank line? I wasn't aware that was part of the style guide. All other PR's I've merged have had the single blank lines

Reply:
Oh really? I would have commented if I noticed. It's not explicitly part of the guide but more consistent with the codebase.

IMO
```
/**
 * @param
 * @param
 * @param
 */
```
looks better and is more consistent with our code than

```
/**
 * @param

 * @param

 * @param
 */
```
even this would be better (which is what I thought you normally did). Difference being the blank lines still have a `*` as part of the comment block. I'm fine with this.

```
/**
 * @param
 *
 * @param
 *
 * @param
 */
```